mqtt/mqtt-broker/src/lib.rs

```diff
@@ -239,7 +240,7 @@ pub enum SystemEvent {
     Shutdown,
     StateSnapshot(StateSnapshotHandle),
     ForceClientDisconnect(ClientId),
-    // ConfigUpdate,
+    AuthorizationUpdate(Box<dyn Any + Send + Sync>),
```

Comment:
Why does it need to be a box of send and sync?

Reply:
Send and Sync need to be implemented so this box can be shared across threads, which it is as a SystemEvent